releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -2,6 +2,50 @@
 
 [Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
 
+## 1.1.0.M6 - December 09, 2019
+
+* Updated support for AAD. System Assigned identity previously required; config store name, and the Contributor role. Now it requires; config store config store name, [client id][client_id], and the App Configuration Data Reader role.
+
+```properties
+spring.cloud.azure.appconfiguration.managed-identity.client-id=[client id]
+spring.cloud.azure.appconfiguration.stores[0].name=[config store name]
+```
+
+* Updated support for AAD. User Assigned identity previously required; config store name, client id, object id, and the Contributor role. Now it required; config store name, client id, and the App Configuration Data Reader role.
+* Added the following Provider to allow users to provide any [Azure TokenCredentials][token_credentials] to the provider for authenticating with App Configuration & Key Vault. When a `TokenCredentialProvider` `@Bean` is made it will be used for authentication.
+
+**Note:** Null is valid value to be returned if only one of the authentication methods needs to be used.
+
+**Note 2:** When using TokenCredentialProvider `spring.cloud.azure.appconfiguration.stores[0].name` still needs to be specified in a configuration file.
+
+```java
+public class MyCredentials implements TokenCredentialProvider {
+
+    @Override
+    public TokenCredential credentialForAppConfig() {
+            return buildCredential();
+    }
+
+    @Override
+    public TokenCredential credentialForKeyVault() {
+            return buildCredential();
+    }
+
+    TokenCredential buildCredential() {
+            return new UsernamePasswordCredentialBuilder()
+                    .clientId("<YOUR_CLIENT_ID>")
+                    .username("<YOUR_USERNAME>")
+                    .password("<YOUR_PASSWORD>")
+                    .build();
+    }
+
+}
+```
+
+* An error is now thrown when more than 1 authentication method is provided.
```

Comment:
What exactly is the error that will be thrown?

I think we should be more specific here. How about this?

* To prevent unintentional results, a {specific error} exception will be thrown if both `spring.cloud.azure.appconfiguration.stores[0].name` and `spring.cloud.azure.appconfiguration.stores[0].connection-string` are provided.